src/Umbraco.Web.UI.Client/src/less/components/overlays.less

```diff
@@ -148,7 +148,14 @@
 
 @media (max-width: 767px) {
     .umb-overlay.umb-overlay-left {
-        margin-left: 60px;
+        margin-left: 61px;
+    }
+}
+
+@media (max-width: 500px) {
+    .umb-overlay.umb-overlay-left {
+        margin-left: 41px;
+        width: auto;
```

Comment:
I found an issue with ´´´width: auto´´´. The overlay calculates how wide it should be and makes indentation based on that calculation. If it gets ```width: auto``` it seems to be confused and makes it very small, but only on some scenarios.

I could trigger the problem if I did a page refresh and then resized the browser to hit the small break point and then opened the user overlay.

I found a possible solution. If we make a calc function like this ```width: ~"(calc(~'100%' - ~'41px'))";``` It seems to work okay and I couldn't trigger the problem again. We could probably do the same for the 767 breakpoint.